.github/workflows/check-ios-build.yml

```diff
@@ -28,10 +28,25 @@ jobs:
           token: ${{ secrets.EXPO_TOKEN }}
 
       - name: Install dependencies
-        run: yarn install
+        run: |
+          yarn install
+          yarn cache clean
+
+      - name: Free up disk space
+        run: |
+          sudo rm -rf ~/Library/Developer/Xcode/DerivedData/*
+          sudo rm -rf /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/*
+          sudo rm -rf ~/Library/Developer/CoreSimulator/Devices/*
+          sudo rm -rf ~/Library/Developer/CoreSimulator/Caches/*
```

Comment:
this seems to have helped the action pass - see the disk usage percentage improvements on this run on a test branch:

![image](https://github.com/user-attachments/assets/c5cff73f-c0aa-42fe-ad24-8b441b1bb0c0)
https://github.com/ephemeraHQ/converse-app/actions/runs/10802863234/job/29965749103